### DISCUSSION TITLE
Bring QuickEmu to Flatpak and Snap if not possible to get in other distro.

### DISCUSSION THREAD

**Discussion by shawny43** [quickemu-project/quickemu#1263] [Ideas]
Soo the idea about it's bring QuickEmu to Flatpak and Snap for better way to install safely the software to any distro. This could solve most problem providen with the package installation and make a quality of life better over updates. Im currently using Bazzite (An Immutable Fedora Silverblue distro) and it's could be cool if there an flatpak install instead of touching the os-tree if we could. 

Reason about flatpak is about safely install apps without scrap the main OS. Most packages could conflicts each others by fault of the package reason when Flatpak or snap don't do that. Instead FP and snap are isolated and each apps got their own package that don't conflict with your OS. 

There some benefits of using Flatpaks and snaps.

- Mostly distro use flatpak or snap has an third-party store provider.
- Apps are sandboxed and less complicated for any user to get better security agaisn't malware.
- Apps with flatpak or snap are package insolated and don't conflicts with the main package OS. 
- Highly compatible with immutable OS such Fedora Silverblue, or VanillaOS has exemple. 

Some disavantage over there.

- Ubuntu could force user to use Snap instead of any other pakcages that include flatpak. (May require to use a terminal to install flatpak.)
- Sandboxed apps doesn't mean your theme in your desktop may apply. (Require some tweaks.)
- Provided with Flathub mostly and poor third-party repos choices towards flatpaks experience. (other packages such Debian have a lot of repos.)

**Comment by flexiondotorg** [comment]
I am going to look into creating a snap of Quickemu. We'd welcome contributions from anyone interested in creating a Flatpak.
